x/ccv/provider/keeper/relay.go

```diff
@@ -220,7 +221,7 @@ func (k Keeper) QueueVSCPackets(ctx sdk.Context) {
 			k.OptInTopNValidators(ctx, consumerId, activeValidators, minPower)
```

Comment:
We don't need to opt in validators here for computing the next validators, right? However, we want to automatically opt in to TopN to not change the logic. Right?

Reply:
Yes exactly. It can be removed but it will require more refactoring. I will create an issue.